client/webserver/site/src/js/markets.ts

```diff
@@ -1270,12 +1346,15 @@ export default class MarketsPage extends BasePage {
     const baseWallet = app().assets[mkt.base.id].wallet
     if (baseWallet.balance.available < mkt.cfg.lotsize) {
       this.setMaxOrder(null)
+      this.updateOrderBttnState()
```

Comment:
Looks like you move `this.updateOrderBttnState` into `setMaxOrder` and reduce duplication.

Reply:
It is a slight duplication, but the intention here was to avoid overloading `maxOrder` with `updateOrderBttnState` calls for the sake of readability.

Reply:
Let me know if you're okay with that or I can optimize this